web/app.js

```diff
@@ -945,7 +949,11 @@ const PDFViewerApplication = {
     }
     // Set the necessary API parameters, using all the available options.
     const apiParams = AppOptions.getAll(OptionKind.API);
-    const params = { ...apiParams, ...args };
+    const params = {
+      ...apiParams,
+      ...args,
+      canvasMaxAreaInBytes: this.externalServices.canvasMaxAreaInBytes,
+    };
```

Comment:
Please re-order this as follows, since the general idea here is that `args` should always be able to override any previous values.
```suggestion
    const params = {
      canvasMaxAreaInBytes: this.externalServices.canvasMaxAreaInBytes,
      ...apiParams,
      ...args,
    };
```